cmd/tke-installer/app/installer/types/types.go

```diff
@@ -84,6 +84,7 @@ type ElasticSearch struct {
 	ReserveDays int    `json:"reserveDays" validate:"required"`
 	Username    string `json:"username"`
 	Password    string `json:"password"`
+	Index       string `json:"index"`
```

Comment:
There is index, but ElasticSearchStorage struct has indices. Is it inconsistent？